### PR TITLE
Changed a default UnitSystem in GB as  metric

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Changed a default UnitSystem in GB  from `imperial` to  `metric`.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
* https://www.worldatlas.com/articles/does-england-use-the-metric-system.html

### CODE REVIEW

File: js/test/units/testunitfmt_usages.js [hunk 1]
```diff
@@ -2732,6 +2750,7 @@ module.exports.testunitfmt_usages = {
 
         var uf = new UnitFmt({
             usage: "storageVolume",
+            measurementSystem: "imperial",
             length: "short",
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_usages.js:2754]: There should be some new tests that test that the usages for en-GB system is metric now except for certain usages. For example, drink volumes are imperial and person weight is imperial, but volume of a gas tank is liters and weight of a car is metric tonnes.

Comment (gouniLee) [reply js/test/units/testunitfmt_usages.js:2754]: I added more test cases with usage. Please re-review them

File: js/test/units/testunitfmt_en_GB.js [hunk 2]
```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
...
+    testUnitFormatWithUsagePersonWeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "kg",
+            amount: 58
+        });
+
+        var uf = new UnitFmt({
+            usage: "personWeight",
+            measurementSystem: "imperial",
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_en_GB.js:353]: These tests should not specify the measurement system. The usage should automatically select the right measures.

File: js/test/units/testunitfmt_en_GB.js [hunk 3]
```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
...
+
+        var uf = new UnitFmt({
+            usage: "babyWeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB",
+            style: "numeric"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "30 lb, 14 oz");
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_en_GB.js:463]: A 30 pound baby?!?!? My God, that mother probably has permanent damage after that birth.

File: js/test/units/testunitfmt_en_GB.js [hunk 4]
```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
...
+        });
+        var str = uf.format(m1);
+        test.equal(str, "30 pounds, 14 ounces");
+        test.done();
+    },
+    testUnitFormatWithUsageDrinkVolume_short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "ml",
+            amount: 1234
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_en_GB.js:488]: Try 569ml for a drink. That should come out as 1 pint.

File: js/test/units/testunitfmt_en_GB.js [hunk 5]
```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
...
+        });
+        var str = uf.format(m1);
+        test.equal(str, "1.09 qt(i)");
+        test.done();
+    },
+    testUnitFormatWithUsageDrinkVolume_Long_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "ml",
+            amount: 1234
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_en_GB.js:504]: same here

File: js/test/units/testunitfmt_en_GB.js [hunk 6]
```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
...
+        });
+
+        var uf = new UnitFmt({
+            usage: "personWeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "128 lb");
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_en_GB.js:358]: For England, this should come out as "9.14286 stone". Person weights are in stone. In the US, it is pounds.

File: js/test/units/testunitfmt_en_GB.js (hunk 2 above)
Comment (gouniLee) [reply js/test/units/testunitfmt_en_GB.js:353]: Without giving a specific measurementSystem value, it returns a metic value and I don't know how to automatically select the right measure..couldn't find the code.  how to select the correct cases unit not follow the default?

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**